sql/core/src/main/scala/org/apache/spark/sql/execution/command/ddl.scala

```diff
@@ -229,10 +230,8 @@ case class AlterTableSetPropertiesCommand(
   extends RunnableCommand {
 
   override def run(sparkSession: SparkSession): Seq[Row] = {
-    val ident = if (isView) "VIEW" else "TABLE"
     val catalog = sparkSession.sessionState.catalog
     DDLUtils.verifyAlterTableType(catalog, tableName, isView)
-    DDLUtils.verifyTableProperties(properties.keys.toSeq, s"ALTER $ident")
```

Comment:
I am wondering if it is possible to not change this part in this part? We can decide if we want to remove it in its individual pr.